src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+function Categories() {
+  return <h1>Categories</h1>;
+}
+
```

Comment:
As specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/3954da2af5b97dd8269f92f236d92c29844679ad/bookstore/project_init_project_wth_comps_v2_0.md?plain=1#L35), kindly include a button with text "Check Status"

Reply:
Thank you will do